csrankings.csv

```diff
@@ -6667,21 +6686,27 @@ Joydeep Biswas,University of Massachusetts Amherst,https://www.cics.umass.edu/pe
 Joydeep Chandra,IIT Patna,https://iitp.ac.in/index.php/departments/computer-center/people/head/705-dr-joydeep-chandra.html,_MaLUpIAAAAJ
 Jozef Gruska,Masaryk University,https://www.muni.cz/en/people/3026,NOSCHOLARPAGE
 Jozsef Csicsvari,IST Austria,https://ist.ac.at/research/research-groups/csicsvari-group,NOSCHOLARPAGE
+João Araújo 0001,Universidade NOVA de Lisboa,http://nova-lincs.di.fct.unl.pt/person/4020,tP9cyo0AAAAJ
 João Batista Neto,USP-ICMC,http://conteudo.icmc.usp.br/pessoas/jbatista,NOSCHOLARPAGE
 João Comba,UFRGS,http://www.inf.ufrgs.br/~comba,NOSCHOLARPAGE
+João Costa Seco,Universidade NOVA de Lisboa,http://nova-lincs.di.fct.unl.pt/person/26,EPFxA8YAAAAJ
 João Dias,Universidade de Lisboa,https://pt-br.facebook.com/public/Joao-Dias/school/fac.ciencias.ul,NOSCHOLARPAGE
 João Dias Pereira,Universidade de Lisboa,http://ihc.fcsh.unl.pt/pt/ihc/investigadores/item/1161-jdpereira,IREJZj4AAAAJ
 João Eduardo Ferreira,USP,http://www.ime.usp.br/~jef,wR2O6I0AAAAJ
 João Garcia,Universidade de Lisboa,www.gsd.inesc-id.pt/~jog,hYb5gDYAAAAJ
 João Henrique de Souza Pereira,UFU,http://www.portal.facom.ufu.br/node/420,NOSCHOLARPAGE
+João Leite,Universidade NOVA de Lisboa,http://nova-lincs.di.fct.unl.pt/person/174,ReF98QcAAAAJ
+João Lourenço,Universidade NOVA de Lisboa,http://nova-lincs.di.fct.unl.pt/person/17,8aN-HtMAAAAJ
 João Luiz Dihl Comba,UFRGS,http://www.inf.ufrgs.br/~comba,NOSCHOLARPAGE
 João Luís Garcia Rosa,USP-ICMC,http://conteudo.icmc.usp.br/pessoas/joaoluis,YsO1rOUAAAAJ
 João M. Lemos,Universidade de Lisboa,https://fenix.tecnico.ulisboa.pt/homepage/ist11886,NOSCHOLARPAGE
 João Madeiras Pereira,Universidade de Lisboa,http://lisboa.academia.edu/Jo%C3%A3oMadeirasPereira,kNDBE9kAAAAJ
 João Marcelo X. N. Teixeira,UFPE,http://www.cin.ufpe.br/voxarlabs,CQNizNEAAAAJ
 João Marcelo Xavier Natario Teixeira,UFPE,http://www.cin.ufpe.br/voxarlabs,CQNizNEAAAAJ
+João Magalhães,Universidade NOVA de Lisboa,http://nova-lincs.di.fct.unl.pt/person/46,P6fBAXMAAAAJ

```

Comment:
Please replace these generic home pages with the actual personal home pages, e.g., http://ctp.di.fct.unl.pt/~jmag/